core/trino-main/src/test/java/io/trino/block/BlockAssertions.java

```diff
@@ -549,8 +549,13 @@ public static Block createLongSequenceBlock(int start, int end)
     public static Block createLongDictionaryBlock(int start, int length)
     {
         checkArgument(length > 5, "block must have more than 5 entries");
+        return createLongDictionaryBlock(start, length, length / 5);
+    }
+
+    public static Block createLongDictionaryBlock(int start, int length, int dictionarySize)
```

Comment:
I think you should change meaning of arguments. Make `length` the actual length of underlying dictionary (that would be consistent with other methods) and make `dictionarySize` length of top-level block

Reply:
Ok Let's keep it as is